src/app/tab3/tab3.page.ts

```diff
@@ -8,33 +10,44 @@ import { ResetPasswordComponent } from './reset-password/reset-password.componen
 @Component({
   selector: 'app-tab3',
   templateUrl: 'tab3.page.html',
-  styleUrls: ['tab3.page.scss']
+  styleUrls: ['tab3.page.scss'],
 })
 export class Tab3Page {
-
-  constructor(private modalController: ModalController) {}
+  constructor(
+    private modalController: ModalController,
+    private _loader: LoaderService,
```

Comment:
Remove underscores. This is deprecated in the new esLint Rules

Reply:
Done